src/posts/2021-07-02-accessibility-seo.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Accessibility and SEO
+description: "SEO and accessibility have a lot more in common than you would think. Many of the same practices that we use to further enable people to enjoy our digital experiences are also great for search engines."
+category: Myth
+author: Cooper Hollmaier
+date: 2021-07-01
+further_reading:
+  - title: "Plain Language Website"
+    url: https://www.plainlanguage.gov/
+    source: United States Government
+tags:
+  - myth
+---
+
+Search engine optimization and accessibility have a lot more in common than you would think.
+
+Many of the same practices that we use to enable everyone to enjoy our digital experiences also open the doors for search engines.
+
+## The intersection of people and search engines
+
+Search engine optimization often gets a bad reputation. In the early years, marketers were adding blocks of keywords into pages by the truck load. Getting to the top of the results by any means necessary was the game. Companies focused on publishing content that was machine-first, not people-first.
+
+The Search Engine Optimization <abbr>SEO</abbr> industry has since evolved. Marketers who choose to build for people over profit, are those that gain increased visibility online. Catering to bots and algorithm changes, by contrast, is often a game of whack-a-mole and traffic gains are short lived.
```

Comment:
```suggestion
The Search Engine Optimization (<abbr>SEO</abbr>) industry has since evolved. Marketers who choose to build for people over profit, are those that gain increased visibility online. Catering to bots and algorithm changes, by contrast, is often a game of whack-a-mole and traffic gains are short lived.
```

Reply:
```suggestion
The Search Engine Optimization <abbr>SEO</abbr> industry has since evolved. Marketers who choose to build for people over profit are those that gain increased visibility online. Catering to bots and algorithm changes, by contrast, is often a game of whack-a-mole and traffic gains are short-lived.
```